python/cuml/decomposition/pca.pyx

```diff
@@ -348,6 +395,9 @@ class PCA(Base):
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
             ndarray, cuda array interface compliant array like CuPy
 
+            sparse array-like (device) shape = (n_samples, n_features)
+            Acceptable formats: cupy.sparse
```

Comment:
I mentioned this below already but we probably want to accept `scipy.sparse` and put it on device if not already.